topics/sequence-analysis/tutorials/ncbi-blast-against-the-madland/tutorial.md

```diff
@@ -0,0 +1,172 @@
+
+
+> <agenda-title></agenda-title>
+>
+> In this tutorial, we will deal with:
+>
+> 1. TOC
+> {:toc}
+>
+{: .agenda}
+
+## Get data
+
+> <hands-on-title> Data Upload </hands-on-title>
+>
+> 1. Create a new history for this tutorial and give it a proper name
+>
+>    {% snippet faqs/galaxy/histories_create_new.md %}
+>    {% snippet faqs/galaxy/histories_rename.md %}
+>
+> 2. Import the file `query.faa` from [Zenodo](https://doi.org/10.5281/zenodo.7524427) 
+>
+>    ```
+>    https://zenodo.org/api/files/40445ead-6429-463c-bfa5-e1fb92095af8/query.faa
+>    ```
+>
+>    {% snippet faqs/galaxy/datasets_import_via_link.md %}
+>   
+>
+{: .hands_on}
+
+> We just imported a FASTA file into Galaxy. Now, the next would be to perfrom the BLAST analysis against MAdLandDB. 
+
+## Perform NCBI Blast+ on Galaxy
+
+>   Since MAdLandDB is the collection of protein sequences, You can perform {% tool [BLASTp](toolshed.g2.bx.psu.edu/repos/devteam/ncbi_blast_plus/ncbi_blastp_wrapper/2.10.1+galaxy2) %} and {% tool [BLASTx](toolshed.g2.bx.psu.edu/repos/devteam/ncbi_blast_plus/ncbi_blastx_wrapper/2.10.1+galaxy2) %} tools.
+
+> <hands-on-title> Similarity search against MAdLand Database </hands-on-title>
+>
+> 1. {% tool [BLASTp](toolshed.g2.bx.psu.edu/repos/devteam/ncbi_blast_plus/ncbi_blastp_wrapper/2.10.1+galaxy2) %} OR {% tool [BLASTx](toolshed.g2.bx.psu.edu/repos/devteam/ncbi_blast_plus/ncbi_blastx_wrapper/2.10.1+galaxy2) %} with the following parameters:
+>    - _"Protein query sequence(s)"_: `Amino acid input sequence` (In case of BLASTp) *OR* 
+>    - _"Translated nucleotide query sequence(s)"_: `Translated nucleotide input sequence` (In case of BLASTx) 
+>    - _"Subject database/sequences"_: `Locally installed BLAST database`
+>    - _"Protein BLAST database"_: `MadLandDB (Genome zoo) plant and algal genomes with a focus on non-seed plants and streptophyte algae (22 Dec 2022)`
+>    - _"Set expectation value cutoff"_: `0.001`
+>    - _"Output format"_:
+>    - In _"Output Options"_: `Tabular (extended 25 columns)` 
+> <img src="../../images/ncbi-blast-against-the-madland/blast-example.png" alt="blast against madland" width="80%">
+>
+{: .hands_on}
+
+## Blast output
+
+> {% icon tool %} The BLAST output will be in tabular format (you can select the desired output format from the drop down menu) and include the following fields : 
+
+>
+
+<table>
+  <tr>
+    <th>Column</th>
+    <th>NCBI name</th>
+    <th>Description</th>
+  </tr>
+  <tr>
+    <td>1</td>
+    <td>qseqid</td>
+    <td>Query Seq-id (ID of your sequence)</td>
+  </tr>
+  <tr>
+    <td>2</td>
```

Comment:
```suggestion
Column | NCBI Name | Description
--- | --- | ---
1 | qseqid | Query Seq-id (ID of your sequence)
```

here you can use a markdown table which we would prefer, if possible.

Reply:
done :)